### PR TITLE
How to create Blank Raw Image User Guide

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: tavni <tavni@redhat.com>

**What this PR does / why we need it**:
A user guide for creation of a blank raw image

**Release note**:

```release-note
A user guide for creation of a blank raw image
```

### CODE REVIEW

File: doc/blank-raw-image.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Prerequesites
+You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+
+## Create Blank Raw Image with DataVolume manifest
+
+Create the following DataVolume manifest (blank-image-datavolume.yaml):
+
+```bash
+apiVersion: cdi.kubevirt.io/v1alpha1
```
Comment (danielerez) [inline doc/blank-raw-image.md]: shouldn't it be v1alpha2?

File: doc/blank-raw-image.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+kind: DataVolume
+metadata:
+  name: blank-image-datavolume
+spec:
+  source:
+      blank: {}
+  pvc:
+    #the storageClassName is optional, if missing we use the default storage class.
```
Comment (danielerez) [inline doc/blank-raw-image.md]: Maybe better to keep using the convention in line 54. I.e. "# Optional: ..."

File: doc/blank-raw-image.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    #storageClassName: "hostpath"
```
Comment (danielerez) [inline doc/blank-raw-image.md]: missing whitespace: "# "

File: doc/blank-raw-image.md (hunk 2 above)
Comment (zvikorn) [reply doc/blank-raw-image.md]: Both DataVolume an PVC were taken from manifest/example.

File: doc/blank-raw-image.md (hunk 3 above)
Comment (zvikorn) [reply doc/blank-raw-image.md]: will do

File: doc/blank-raw-image.md (hunk 1 above)
Comment (zvikorn) [reply doc/blank-raw-image.md]: Nop.

**Comment by igoihman** [comment]
failures not related to this change

**Pull request merged**